src/test/java/org/isf/patient/test/Tests.java

```diff
@@ -232,6 +278,448 @@ public void testMergePatientHistory() throws Exception {
+		Integer code = _setupTestPatient(false);
+		Patient patient = patientIoOperation.getPatient(code);
+		patient.setFirstName("someNewFirstName");
+		Patient updatedPatient = patientBrowserManager.savePatient(patient);
+		assertThat(updatedPatient).isNotNull();
+		assertThat(updatedPatient.getFirstName()).isEqualTo(patient.getFirstName());
+	}
+
+	@Test
+	public void testMgrDeletePatient() throws Exception {
+		Integer code = _setupTestPatient(false);
+		Patient patient = patientIoOperation.getPatient(code);
+		assertThat(patientBrowserManager.deletePatient(patient)).isTrue();
+	}
+
+	@Test
+	public void testMgrIsNamePresent() throws Exception {
+		Integer code = _setupTestPatient(false);
+		Patient foundPatient = patientIoOperation.getPatient(code);
+		assertThat(patientBrowserManager.isNamePresent(foundPatient.getName())).isTrue();
+	}
+
+	@Test
+	public void testMgrIsNamePresentNotFound() throws Exception {
+		assertThat(patientBrowserManager.isNamePresent("someNameWeAreSureDoesNotExist")).isFalse();
+	}
+
+	@Test
+	public void testMgrGetNextPatientCode() throws Exception {
+		Integer code = _setupTestPatient(false);
+		Integer max = patientBrowserManager.getNextPatientCode();
+		assertThat((code + 1)).isEqualTo(max);
+	}
+
+	@Test
+	public void testMgrGetMartialList() throws Exception {
+		resetHashMaps();
+		String[] maritalDescriptionList = patientBrowserManager.getMaritalList();
+		assertThat(maritalDescriptionList).isNotEmpty();
+	}
+
+	@Test
+	public void testMgrGetMartialTranslated() throws Exception {
+		resetHashMaps();
+		// TODO: if resource bundles are made avaiable in core then the values being compared will need to change
+		assertThat(patientBrowserManager.getMaritalTranslated(null)).isEqualTo("angal.patient.maritalstatusunknown");
+		assertThat(patientBrowserManager.getMaritalTranslated("someKeyNotInTheList")).isEqualTo("angal.patient.maritalstatusunknown");
+		assertThat(patientBrowserManager.getMaritalTranslated("married")).isEqualTo("angal.patient.maritalstatusmarried");
+	}
+
+	@Test
+	public void testMgrGetMartialKey() throws Exception {
+		resetHashMaps();
+		// TODO: if resource bundles are made avaiable in core then the values being compared will need to change
+		assertThat(patientBrowserManager.getMaritalKey(null)).isEqualTo("undefined");
+		assertThat(patientBrowserManager.getMaritalKey("someKeyNotInTheList")).isEqualTo("undefined");
+		assertThat(patientBrowserManager.getMaritalKey("angal.patient.maritalstatusmarried")).isEqualTo("married");
+	}
+
+	@Test
+	public void testMgrGetProfessionList() throws Exception {
+		resetHashMaps();
+		String[] maritalDescriptionList = patientBrowserManager.getProfessionList();
+		assertThat(maritalDescriptionList).isNotEmpty();
+	}
+
+	@Test
+	public void testMgrGetProfessionTranslated() throws Exception {
+		resetHashMaps();
+		// TODO: if resource bundles are made avaiable in core then the values being compared will need to change
```

Comment:
indeed "bundles" in core is something we have to think about (one can see OP-291)